modules/ebi-metagenomics/dram/distill/main.nf

```diff
@@ -0,0 +1,54 @@
+process DRAM_DISTILL {
+    tag "$meta.id"
+    label 'process_high'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/dram:1.3.5--pyhdfd78af_0':
+        'quay.io/biocontainers/dram:1.3.5--pyhdfd78af_0' }"
+
+    containerOptions="--bind ${moduleDir}/tests/fixtures/dram_distill_dbs/:/data/ --bind ${moduleDir}/tests/fixtures/dram_distill_dbs/CONFIG:/usr/local/lib/python3.10/site-packages/mag_annotator/CONFIG"
+
+    input:
+    tuple val(meta), path(dram_input)
+
+    output:
+    tuple val(meta), path("*_dram.html"), emit: distill_out_html // removed "optional" e.g. we don't launch the module when missing input files
+    tuple val(meta), path("*_dram.tsv") , emit: distill_out_tsv
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def VERSION = '1.3.5' // WARN: dram has no option to print the tool version. This is the container version
+
+    """
+    DRAM.py \\
+        distill \\
+        -i ${dram_input}  \\
+        -o dram_out
```

Comment:
not sure if DRAM has any of these, but if it can have other parameters that are optional, this is where you should add $args as an extra line, makes it easy for users to add any extra arguments if they wish